pallets/validators-manager/src/benchmarking.rs

```diff
@@ -10,102 +10,129 @@ use super::*;
 use pallet_parachain_staking::{Currency, Pallet as ParachainStaking};
 use pallet_session::Pallet as Session;
+use secp256k1::{PublicKey, SecretKey};
 use sp_avn_common::eth_key_actions::decompress_eth_public_key;
-use sp_core::ecdsa::Public;
-
-fn setup_validators<T: Config>(
-    number_of_validator_account_ids: u32,
-) -> Vec<Validator<<T as pallet_avn::Config>::AuthorityId, T::AccountId>> {
-    let mnemonic: &str =
-        "basic anxiety marine match castle rival moral whisper insane away avoid bike";
-    let mut validators: Vec<Validator<<T as pallet_avn::Config>::AuthorityId, T::AccountId>> =
-        Vec::new();
-    for i in 0..number_of_validator_account_ids {
-        let account = account("dummy_validator", i, i);
-        let key =
-            <T as avn::Config>::AuthorityId::generate_pair(Some(mnemonic.as_bytes().to_vec()));
-        validators.push(Validator::new(account, key));
-    }
+use sp_core::{ecdsa::Public, H512};
 
-    // Setup sender account id and key
-    let sender_index = validators.len() - (1 as usize);
-    let sender: Validator<T::AuthorityId, T::AccountId> = validators[sender_index].clone();
-    let mut account_bytes: [u8; 32] = [0u8; 32];
-    account_bytes
-        .copy_from_slice(&hex!("b41f90b123b66c18f0f869f3b9ae8a09d118419f8736240fcc7b8256517cc233"));
-    let account_id = T::AccountId::decode(&mut &account_bytes.encode()[..]).unwrap();
-    validators[sender_index] = Validator::new(account_id, sender.key);
+// Derived from [6u8;32] private key
+const RESIGNING_COLLATOR_ETHEREUM_PUBLIC_KEY: [u8; 33] =
+    hex!["03f006a18d5653c4edf5391ff23a61f03ff83d237e880ee61187fa9f379a028e0a"];
+
+const NEW_COLLATOR_ETHEREUM_PUBLIC_KEY: [u8; 33] =
+    hex!["03f171af36531200540b2badee5ed581b0a51f4e4a1a995025e149b9721b050074"];
+
+// Derived from [7u8;32] private key
+const VOTER_SENDER_ETHEREUM_COLLATOR_PUBLIC_KEY: [u8; 33] =
+    hex!["02989c0b76cb563971fdc9bef31ec06c3560f3249d6ee9e5d83c57625596e05f6f"];
+
+const MINIMUM_ADDITIONAL_BENCHMARKS_VALIDATORS: usize = 2;
+
+const MNEMONIC: &str =
+    "basic anxiety marine match castle rival moral whisper insane away avoid bike";
+
+fn generate_resigning_collator_account_details<T: Config>(
+) -> (T::AccountId, <T as pallet_avn::Config>::AuthorityId, Public) {
+    let key = <T as avn::Config>::AuthorityId::generate_pair(Some(MNEMONIC.as_bytes().to_vec()));
 
-    // Setup resigner account id and key
-    let resigner: Validator<T::AuthorityId, T::AccountId> = validators[1].clone();
     let mut resigner_account_bytes: [u8; 32] = [0u8; 32];
     resigner_account_bytes
-        .copy_from_slice(&hex!("1ed1aadead9704b693af012a9f24e1f00dc7e2a0b4eb99f9e0bc0c35a8d20223"));
-    let resigner_account_id =
-        T::AccountId::decode(&mut &resigner_account_bytes.encode()[..]).unwrap();
-    validators[1] = Validator::new(resigner_account_id, resigner.key);
+        .copy_from_slice(&hex!("0e08c08252ef2a5cbeb5d6a0dcaa1b8354aded57d603cdc4adc5e27709a6904f"));
```

Comment:
Can we make this a constant? How does this relate to the key in line 47?
Also, could we make clear in the variable name (`resigner_account_bytes`) what chain this account is in? And is this basically the account **public** key?